### PR TITLE
Add specs

### PR DESCRIPTION
**Pull request by kristjan**
@apartmentlist/supply

See eddiezane/sendgrid-actionmailer#2 for summary. Not sure how long it might take to merge upstream.

### CODE REVIEW

**Comment by quarterdome** [comment]
So what is the piece of functionality you need to add to sendgrid-actionmailer?

**Comment by kristjan** [comment]
Small piece in the next PR. Their API `from` only supports email addresses, and there's a separate `fromname` to say "Apartment List" or something. Going to parse that out of the `from` coming in on the `Mail` object.

File: spec/spec-helper.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Requires supporting ruby files with custom matchers and macros, etc,
+# in spec/support/ and its subdirectories.
+Dir["#{File.dirname(__FILE__)}/support/**/*.rb"].each { |f| require f }
```
Comment (tcollier) [inline spec/spec-helper.rb:3]: This won't match any files right now. Unless you have a need for it in an imminent PR, you can just remove this line.

**Comment by tcollier** [comment]
LGTM

File: spec/spec-helper.rb (hunk 1 above)
Comment (kristjan) [inline spec/spec-helper.rb:3]: I removed a bunch of stuff but figured this was a common enough pattern to leave it for later. What do you think?

**Pull request merged**